src/Core/Mesh/TriangleMesh.hpp

```diff
@@ -150,8 +168,10 @@ class TriangleMesh {
         m_normalsHandle = m_vertexAttribs.addAttrib<NormalAttribHandle::value_type>( "in_normal" );
     }
 
-    /// Access the attribute manager
+    /// Access the attribute manager.
     const AttribManager& attribManager() const { return m_vertexAttribs; }
```

Comment:
The attribute cannot be visible as we restrict its interface in `TriangleMesh` (in_position, ...).
This method needs to be removed.